### PR TITLE
docs(Forms): improves docs for Value.Number & Value.Currency

### PR DESCRIPTION
**Pull request by langz**
Can be demoed here:
- [Currency](https://eufemia-git-chore-improve-currency-docs-eufemia.vercel.app/uilib/extensions/forms/Value/Currency/properties/)
- [Number](https://eufemia-git-chore-improve-currency-docs-eufemia.vercel.app/uilib/extensions/forms/Value/Number/properties/)

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Value/Number/NumberDocs.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { PropertiesTableProps } from '../../../../shared/types'
+import { NumberFormatPropertiesCamelCase } from '../../../../components/number-format/NumberFormatDocs'
 
 export const NumberProperties: PropertiesTableProps = {
+  ...NumberFormatPropertiesCamelCase,
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/extensions/forms/Value/Number/NumberDocs.tsx:5]: Here we need to hand pick what we think should be supported. I think we only should show the "currency" relevant props. And not show e.g. `nin`.

**Review by tujoworker — APPROVED** [review]
Nice work 🙏

I think we need to adjust what prop we show for currency.

File: packages/dnb-eufemia/src/extensions/forms/Value/Number/NumberDocs.tsx (hunk 1 above)
Comment (langz) [reply packages/dnb-eufemia/src/extensions/forms/Value/Number/NumberDocs.tsx:5]: _What we think should be supported?_

Isn't all `Value.Number` properties supported in `Value.Currency`? Especially from a [TS perspective](https://github.com/dnbexperience/eufemia/blob/aac721b63daadb9cf8dbf39baf5fbe7d3717eb1f/packages/dnb-eufemia/src/extensions/forms/Value/Currency/Currency.tsx#L13).

If we want to change the supported properties, shouldn't we change the TS as well as in the property docs?
Or can/should these be different?

Comment (langz) [reply packages/dnb-eufemia/src/extensions/forms/Value/Number/NumberDocs.tsx:5]: Do we have any suggestions on which subset of props from its available ts props(NumberValueProps), that we want to display?

Comment (langz) [reply packages/dnb-eufemia/src/extensions/forms/Value/Number/NumberDocs.tsx:5]: I'll open a new PR with my own suggestions, and then we can take it further from there 👍

**Pull request merged**